salt/modules/virt.py

```diff
@@ -4663,7 +4669,12 @@ def cpu_baseline(full=False, migratable=False, out='libvirt', **kwargs):
     return cpu.toxml()
 
 
-def network_define(name, bridge, forward, **kwargs):
+def network_define(name,
+                   bridge,
+                   forward,
+                   ipv4_config: None,
```

Comment:
I see Python3 type checking :trollface: Would be a nice to have, but Python2 won't like this. I do appreciate the type checking but, maybe this will serve just as well: 
```
:param dict ipv4_config:
    Describes the IP v4 setup like IP range and a possible 
    DHCP configuration. The structure is documented in 
    net-define-ip_.
```` 
..
I'm actually a huge fan of Sphinx style:
```
   :param ipv4_config: Describes the IP v4 setup like IP range 
        and a possible DHCP configuration.
        The structure is documented in net-define-ip_.
   :type ipv4_config: dict or None
```

Reply:
I suspect that meant to be `ipv4_config=None` 😝

Reply:
yeah, and that is fixed by now in the PR ;)